lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -40,6 +45,8 @@ class LineChartPainter extends AxisChartPainter {
     });
 
     drawTitles(canvas, viewSize);
+
+    drawAnnotation(canvas, viewSize);
```

Comment:
dude, this is not annotation in a generic definition, look at the problem as a generic problem, we should name it extraLines to draw extra lines on the chart, then you can use it for your problem to draw avg line, this is not related to the FL Chart.

Reply:
Residual naming from the original work when I first started. Renaming.